next-app/src/routes/password/change/+page.server.ts

```diff
@@ -0,0 +1,33 @@
+import { sf, type Rpc } from '$lib/server/sf'
+import { fetch_current_user } from '$lib/server/user'
+import { fail, redirect } from '@sveltejs/kit'
+import type { RequestEvent } from '../$types'
+
+export const actions = {
+  default: async ({ request }: RequestEvent) => {
+	const submissions = await request.formData()
+
+	const new_password = submissions.get('new_password')
+	const new_password_confirm = submissions.get('new_password_confirm')
+
+	if (! new_password) {
```

Comment:
```suggestion
	if (!new_password?.trim()) {
```

Reply:
I wasn't familiar with the use of `?` until I looked it up - it is a pre-check for null/undefined on the object.

Reply:
it might time for us to integrate zxcvbn, trim this feels like the same sort of thing as making sure the provided password is a good one.  I'm not immediately inclined to get rid of those trailing spaces in case it was intentional... I guess we'll need more discussion here.

Reply:
I don't know if zxcvbn is still a good choice for password strength these days...would be good to re-evaluate.

Reply:
I think that's a really good point @megahirt ... maybe we should abandon the idea then.